From fa2e6505f964d3c7b5e4b8c6679404843f15ebb3 Mon Sep 17 00:00:00 2001
From: Sandeep Gutta <sangutta@codeaurora.org>
Date: Wed, 24 Jan 2018 17:03:12 +0530
Subject: [PATCH 4/6] Fix SIM refresh issue

Below registrant is missed in IccRecords and due to that UNSOL REFRESH File update
event is not received to IccRecords.

mCi.registerForIccRefresh()

Added above registrent in IccRecords file to receive UNSOL REFRESH events.

Change-Id: Ife43ad4ef08c860f8614d7c0f8ee61d83d92d44f
CRs-Fixed: 2176288
---
 .../com/android/internal/telephony/uicc/UiccController.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/com/android/internal/telephony/uicc/UiccController.java b/src/java/com/android/internal/telephony/uicc/UiccController.java
index 0e2b9692e..b5127e129 100644
--- a/src/java/com/android/internal/telephony/uicc/UiccController.java
+++ b/src/java/com/android/internal/telephony/uicc/UiccController.java
@@ -353,7 +353,7 @@ public class UiccController extends Handler {
         }
 
         // The card status could have changed. Get the latest state.
-        mCis[index].getIccCardStatus(obtainMessage(EVENT_GET_ICC_STATUS_DONE));
+        mCis[index].getIccCardStatus(obtainMessage(EVENT_GET_ICC_STATUS_DONE, index));
     }
 
     private boolean isValidCardIndex(int index) {
-- 
2.17.1

